### ISSUE TITLE
Not working on Windows WSL2

### ISSUE THREAD

**Issue by loco1842** [stef-k/laravel-goto-controller#20]
Hello.
I have a problem while working on windows wsl2 like the image below. It does not return the correct path when it works on wsl2.
![image](https://user-images.githubusercontent.com/22882819/101462054-47e04d00-396e-11eb-82ae-5d6d785e5f7a.png)

**Comment by stef-k** [comment]
Yes, that is correct, the extension currently does not work on  WSL.
